incubator/fluentd/templates/daemonset.yaml

```diff
@@ -49,7 +49,7 @@ spec:
               fi; if [ -z "$(ls -A /var/log/fluentd-buffers-papertrail)" ]; then
                 echo "/var/log/fluentd-buffers-papertrail is empty. There are no buffers...I think that is good?";
                 exit 0;
-              fi; if [[ -z $(find /var/log/fluentd-buffers-papertrail -mmin -${LIVENESS_THRESHOLD_MINUTES}) ]]; then
+              fi; if [ -n "$(find /var/log/fluentd-buffers-papertrail -mmin ${LIVENESS_THRESHOLD_MINUTES} -name *.log)" ]; then
```

Comment:
Is the flip from `-z` to `-n` intentional?

Reply:
Yep. The intent is to be `true` if the results from a search for a file that was last modified 15 minutes ago returns a string with a length > 0.